modules/install-sensor-roxctl.adoc

```diff
@@ -20,13 +20,21 @@ endif::[]
 
 To monitor a cluster, you must deploy Sensor.
 You must deploy Sensor into each cluster that you want to monitor.
-The following steps describe adding Sensor by using the {product-title-short} portal.
+This installation method is also called the manifest installation method.
+To perform an installation by using the manifest installation method:
+* Use the {product-title-short} web portal to download the cluster bundle, then unzip and run the sensor script.
```

Comment:
This does not seem to turn into bullet points in [preview](https://71919--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-secured-cluster-ocp#install-sensor-roxctl_install-secured-cluster-ocp)

More importantly, I think these two bullets describe alternative approaches to installation. Only one of them should be used for a given secured cluster, not both, line 24 should probably make this more clear 🤔